modules/nf-core/busco/download/main.nf

```diff
@@ -0,0 +1,48 @@
+process BUSCO_DOWNLOAD {
+    tag "$lineage"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/busco:5.8.2--pyhdfd78af_0':
+        'biocontainers/busco:5.8.2--pyhdfd78af_0' }"
+
+    input:
+    val lineage
+
+    output:
+    path "busco_downloads", emit: download_dir
+    path "versions.yml"   , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${lineage}"
+    """
+    busco \\
+        --download $lineage \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        busco: \$( busco --version 2>&1 | sed 's/^BUSCO //' )
+    END_VERSIONS
+    """
+
+    stub:
+    def args = task.ext.args ?: ''
+    """
+    echo "busco \\
+        --download $lineage \\
+        $args"
```

Comment:
I don't think this is necessary. You can simply keep the mkdir below.

Reply:
Ok, I removed the code.